lua/telescope/builtin/lsp.lua

```diff
@@ -184,6 +187,25 @@ lsp.workspace_symbols = function(opts)
   }):find()
 end
 
+lsp.diagnostics = function(opts)
+    local locations = utils.diagnostics_to_tbl(opts)
+    if vim.tbl_isempty(locations) then return end
```

Comment:
Maybe we also wanna `print('No diagnostics found')` or so. Might be better if we give people some feedback that things are happening